ocaml/xapi/xapi_pusb.ml

```diff
@@ -29,8 +29,14 @@ let create ~__context ~uSB_group ~host ~other_config ~path
   debug "PUSB ref='%s' created" (Ref.string_of pusb);
   pusb
 
+let host_ref : [`host] Ref.t ref = ref Ref.null
```

Comment:
This is a global variable. Can it contain anything other than the reference for localhost? And if not, how is this interacting with threads?

Reply:
no need host_ref here now.